extensions/wikia/PortabilityDashboard/SpecialPortabilityDashboardController.php

```diff
@@ -19,7 +19,7 @@ public function index() {
 		$langFilter = $this->getVal( self::LANGUAGE_FILTER_QS_PARAM, '' );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)